src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int LENGTH = 10;
+    private Object[] keys;
+    private Object[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = new Object[LENGTH];
+        this.values = new Object[LENGTH];
```

Comment:
we should use `this` only if we have the same naming for field and input parameter or local variable
```suggestion
        keys = new Object[LENGTH];
        values = new Object[LENGTH];
```